<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSmartIndentCommon</class>
 <widget class="QDialog" name="DialogSmartIndentCommon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Common Smart Indent Macros</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Common Definitions for Smart Indent Macros</string>
     </property>
     <property name="buddy">
      <cstring>editCode</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editCode">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text">
        <string>Appl&amp;y</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok-apply">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCheck">
       <property name="text">
        <string>Chec&amp;k</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore">
       <property name="text">
        <string>Restore D&amp;efaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogSmartIndentCommon</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
